package.json

```diff
@@ -5,7 +5,10 @@
   "author": "Fluid Project",
   "bugs": "https://issues.fluidproject.org/browse/FLOE/component/11615",
   "homepage": "http://www.fluidproject.org/",
-  "dependencies": {},
+  "dependencies": {
+      "d3":"latest",
+      "flocking":"https://github.com/colinbdclark/Flocking"      
+  },
   "licenses": [
```

Comment:
This syntax is no longer supported in current npm: https://docs.npmjs.com/files/package.json#license